### ISSUE TITLE
Build system: CMake namespace

### ISSUE THREAD

**Issue by kintel** [elalish/manifold#936]
See https://github.com/elalish/manifold/blob/ced53e210972adee15edade7c7308fcf2777f707/bindings/c/CMakeLists.txt#L36
```
install(TARGETS ${PROJECT_NAME} EXPORT manifoldTargets)
```

I believe a `NAMESPACE Manifold` or smth. should be added here to correctly namespace the manifold sub-libraries according to the recommendation given as part of https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html#a-sample-find-module

The end goal would be for client projects to be able to do:
```
find_package(manifold REQUIRED)
target_link_libraries(MyApp Manifold::manifold Manifold::polygons)
```

**Comment by pca006132** [comment]
Sounds good to me.

**Comment by elalish** [comment]
Yes, always happy to follow the recommendations. @kintel do you want to make a PR?

**Comment by kintel** [comment]
I'll look into it in a bit; need to set up an e2e test first using an externally built manifold in OpenSCAD.

**Comment by pca006132** [comment]
Addressed in #961.
